src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,21 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "13, Computing Drive, COM1-02-06, 117417";
+    private static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    private static final String SEPARATOR = ",";
+
+    private static final int PARTS_COUNT = 4;
+    private static final int PART_BLOCK      = 0;
```

Comment:
Good use of constants to remove magic numbers.
:+1:  for naming them appropriately following the convention.